cli.sh

```diff
@@ -978,30 +988,30 @@ add_shared_hook_repo() {
 #   the global or local list.
 #####################################################
 remove_shared_hook_repo() {
-    SET_SHARED_GLOBAL=
+    SET_SHARED_TYPE="--shared"
     SHARED_REPO_URL=
 
-    case "$1" in
-    "--global")
-        SET_SHARED_GLOBAL=1
-        SHARED_REPO_URL="$2"
-        ;;
-    "--local")
-        SET_SHARED_GLOBAL=
+    if echo "$1" | grep -qE "\-\-(shared|local|global)"; then
+        SET_SHARED_TYPE="$1"
         SHARED_REPO_URL="$2"
-        ;;
-    *)
+    else
         SHARED_REPO_URL="$1"
-        ;;
-    esac
+    fi
 
     if [ -z "$SHARED_REPO_URL" ]; then
-        echo "! Usage: \`git hooks shared remove [--global|--local] <git-url>\`" >&2
+        echo "! Usage: \`git hooks shared add [--shared|--local|--global] <git-url>\`" >&2
```

Comment:
Erm... this is the `remove_shared_hook_repo` function here?
```suggestion
        echo "! Usage: \`git hooks shared remove [--shared|--local|--global] <git-url>\`" >&2
```